web-api/Dockerfile

```diff
@@ -2,7 +2,9 @@ FROM debian:bookworm-slim
 ARG api
 LABEL api="$api"
 WORKDIR /app
-#renamed as $api doesn't work in ENTRYPOINT
-COPY ./"$api" ./server.bin
+# copy everything in the cwd (bin and assets)
+COPY ./ ./
+# renamed as $api doesn't work in ENTRYPOINT
+RUN mv ./"$api" ./server.bin
```

Comment:
:question: 
Is it worth it to have a parameter only to rename a file? When we generate the docker context we can already name the binary as server.

Reply:
i can remove it

Reply:
done. was a good idea. it makes the pulling faster